docker-compose.yml

```diff
@@ -1,24 +1,56 @@
 version: '3.7'
 
 services:
-  council-daemon:
-    image: lidofinance/lido-council-daemon@sha256:9943ed50556fc8bf75c1179023c33dc30ef9efc6ee4f4f29bfed2fc5bc23ad64
+  keys_api_service_db:
+    image: postgres:14-alpine
+    container_name: keys_api_service_db
+    restart: unless-stopped
+    environment:
+      - POSTGRES_DB=${KEYS_API_DB_NAME}
+      - POSTGRES_USER=${KEYS_API_DB_USER}
+      - POSTGRES_PASSWORD=${KEYS_API_DB_PASSWORD}
+    ports:
+      - ${KEYS_API_DB_PORT}:5432
+    volumes:
+      - ./.volumes/pgdata-${CHAIN_ID}/:/var/lib/postgresql/data
+
+  keys_api_service_api:
+    image: lidofinance/lido-keys-api@sha256:996476f3c272fd74ba89b8e46ec2510bc5c07831ebb276ad661ddb00971f1a21
+    container_name: keys_api_service_api
     ports:
-      - "3000:3000" # port is used for prometheus metrics
+      - '127.0.0.1:${KEYS_API_PORT}:3001'
     environment:
-      - "PORT=3000"
-      - "LOG_LEVEL=debug"
-      - "LOG_FORMAT=simple"
-      - "RPC_URL=<rpc url>"
-      - "KAFKA_SSL=true"
-      - "KAFKA_SASL_MECHANISM=plain"
-      - "KAFKA_USERNAME=<kafka user>"
-      - "KAFKA_PASSWORD=<kafka password>"
-      - "KAFKA_BROKER_ADDRESS_1=<kafka address>"
-      - "BROKER_TOPIC=defender"
-      - "WALLET_PRIVATE_KEY"
+      - PORT=3001
+      - LOG_LEVEL=${LOG_LEVEL}
+      - LOG_FORMAT=${LOG_FORMAT}
+      - CHAIN_ID=${CHAIN_ID}
+      - PROVIDERS_URLS=${RPC_URL}
+      - VALIDATOR_REGISTRY_ENABLE=false
+      - DB_NAME=${KEYS_API_DB_NAME}
+      - DB_PORT=5432
+      - DB_HOST=keys_api_service_db
+      - DB_USER=${KEYS_API_DB_USER}
+      - DB_PASSWORD=${KEYS_API_DB_PASSWORD}
+    depends_on:
+      - keys_api_service_db
 
-    # volumes can be omitted if needed
+  council-daemon:
+    image: lidofinance/lido-council-daemon:dev
```

Comment:
use sha256 instead

Reply:
image is not ready, I will edit it after deploy